src/backend/distributed/transaction/commit_protocol.c

```diff
@@ -197,6 +260,8 @@ CommitRemoteTransactions(List *connectionList, bool stopOnFailure)
 											  command->data, nodeName, nodePort)));
 				}
 			}
+
+			ereport(DEBUG2, (errmsg("sent COMMIT PREPARED over connection %ld", connectionId)));
```

Comment:
no, but I though that distinguishing between matching commits/rollbacks wouldn't hurt anyone.